pkgs/servers/nitter/nitter-version.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/src/views/about.nim b/src/views/about.nim
+index e7e8de9..54a6050 100644
+--- a/src/views/about.nim
++++ b/src/views/about.nim
+@@ -3,10 +3,8 @@ import os, strformat
+ import karax/[karaxdsl, vdom]
+ 
+ const
+-  date = staticExec("git show -s --format=\"%cd\" --date=format:\"%Y.%m.%d\"")
+-  hash = staticExec("git show -s --format=\"%h\"")
```

Comment:
is there a reason to patch all 4 lines rather than patching only date and hash?

```diff
-  date = staticExec("git show -s --format=\"%cd\" --date=format:\"%Y.%m.%d\"")
-  hash = staticExec("git show -s --format=\"%h\"")
+  date = "@version@"
+  hash = "@rev@"
```

Reply:
I wanted to also patch `link`, so that it would handle more cases where the `src` was overridden. Since each variable is thus just used once, thought that I might as well inline them into the `version` line.